service/src/main/java/com/carhouse/service/exception/NotFoundException.java

```diff
@@ -0,0 +1,16 @@
+package com.carhouse.service.exception;
+
+/**
+ * The custom exception.
+ * Throws when resource not found
+ */
+public class NotFoundException extends RuntimeException {
```

Comment:
think Java already has something like this exception

Reply:
Ok, I've found exception similar to my custom exception (javassist.NotFoundException) and replaced on it